src/main/scala/rx/lang/scala/Notification.scala

```diff
@@ -73,7 +73,7 @@ sealed trait Notification[+T] {
  * import Notification._
  * Observable(1, 2, 3).materialize.subscribe(n => n match {
```

Comment:
Could you also change `Observable(1, 2, 3)` to `Observable.just(1, 2, 3)`? `Observable(1, 2, 3)` does not work now.